DemoBookApp/Models/AuthorModel.cs

```diff
@@ -12,8 +12,9 @@ public Guid Id
         [Required(ErrorMessage = "Name is required")]
         [StringLength(30, MinimumLength = 3, ErrorMessage = "Name must be within 3-30 characters")]
         public string Name { get; set; }
+
         [Required(ErrorMessage ="Gender is required")]
-        public int Gender;
+        public int Gender { get; set; }= 0; 
```

Comment:
why we are storing default value in gender?

Reply:
My bad, corrected it